extension/src/shared/reearth/types/reearthPluginAPIv2/events.ts

```diff
@@ -0,0 +1,79 @@
+// import {
```

Comment:
Can we remove this file?

Reply:
Ah sorry forgot clean up this.

Reply:
Ahh? I think it's aleady removed...But i don't know why you can see this file still...